tests/cursor/bug1839-004.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+PHPC-1839: Non-referenced interned string in typeMap
```

Comment:
What makes this test "non-referenced"? Should this test be titled: "Referenced, local, interned string in typeMap"?